client/state/sites/selectors/get-jetpack-computed-attributes.js

```diff
@@ -1,7 +1,6 @@
 /**
```

Comment:
Hmm, this whole module doesn't seem like a particularly good pattern. Thank you for removing one of the methods!

Reply:
Yes, the computed attributes are a leftover from the pre-Redux times when Sites were Flux objects and they had methods.